core/Command/Maintenance/Mimetype/GenerateMimetypeFileBuilder.php

```diff
@@ -30,17 +30,15 @@
 class GenerateMimetypeFileBuilder {
 	/**
 	 * Generate mime type list file
-	 * @param $aliases
+	 *
+	 * @param array $aliases
 	 * @return string
 	 */
 	public function generateFile(array $aliases): string {
 		// Remove comments
-		$keys = array_filter(array_keys($aliases), function ($k) {
-			return $k[0] === '_';
-		});
-		foreach ($keys as $key) {
-			unset($aliases[$key]);
-		}
+		$aliases = array_filter($aliases, static function ($key) {
+			return !($key === '' || $key[0] === '_');
+		}, ARRAY_FILTER_USE_KEY);
```

Comment:
```suggestion
		$aliases = array_filter(
			$aliases, 
			fn (string $key) => !($key === '' || $key[0] === '_'), 
			ARRAY_FILTER_USE_KEY);
```
Why did you set function as static?

Reply:
I'm using Php Inspections (a plugin for PHPStorm). It's a recommendation to declare closures static when possible. They say: better scoping; in some cases can improve performance.

The arrow function is a good suggestion thanks :+1: But I have to modify the PR with the backports then.